apps/finance/app/src/components/NewTransfer/Deposit.js

```diff
@@ -158,17 +159,23 @@ class Deposit extends React.Component {
         tokenDataFallback(address, 'decimals', network.type) || '0'
       const symbolFallback =
         tokenDataFallback(address, 'symbol', network.type) || ''
-
-      combineLatest(token.decimals(), token.symbol())
+      const tokenSymbol = getTokenSymbol(app,address)
+      if (tokenSymbol){
+        combineLatest(token.decimals(), token.symbol())
         .first()
         .subscribe(
-          ([decimals = decimalsFallback, symbol = symbolFallback]) =>
+          ([decimals = decimalsFallback, symbol]) =>
```

Comment:
We are not using `symbol` anymore, so we can probably remove it (and the `token.symbol()` call).